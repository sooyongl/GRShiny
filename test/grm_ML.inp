data: file is grm_dt.csv;
variable:
  names are a1-a4;
  categorical = a1-a4;

analysis:
  estimator = ML;
  
model:

  F1 by a1*
        a2-a4;
  F1@1;
  [F1@0;
  
  
  