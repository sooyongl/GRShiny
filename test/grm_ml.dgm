VERSION 1.1.0
INPUT
DATA:
 file is grm_dt.csv;
VARIABLE:
  names are a1-a4;
  categorical = a1-a4;
ANALYSIS:
  estimator = ML;
MODEL:
  F1 by a1-a4* (L1-L4);
   F1* (FVAR);
  [F1*] (FMEAN);
  [a1$1-a4$1] (T1a1-T1a4);
  [a1$2-a4$2] (T2a1-T2a4);
INPUT_END
2
0
0
0
ML
1
GENERAL
0
4
1
0
a1 a2 a3 a4 f1
1
-1 -1 -1 -1 0
0
1
f1
a1 a2 a3 a4
1 1 f1 a1    0.893    0.161
91 1 f1 a1 l1
1 1 f1 a2    1.852    0.354
91 1 f1 a2 l2
1 1 f1 a3    1.078    0.181
91 1 f1 a3 l3
1 1 f1 a4    1.394    0.233
91 1 f1 a4 l4
5 1 f1    1.000    0.000
95 1 f1 fvar
