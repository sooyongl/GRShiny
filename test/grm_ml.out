Mplus VERSION 8.6
MUTHEN & MUTHEN
01/23/2022   7:56 PM

INPUT INSTRUCTIONS

  data: file is grm_dt.csv;
  variable:
    names are a1-a4;
    categorical = a1-a4;

  analysis:
    estimator = ML;
    parameterization = DELTA;

  model:

    F1 by a1*
          a2-a4;
    F1@1;
    [F1@0;






*** WARNING in ANALYSIS command
  PARAMETERIZATION=DELTA is not allowed for TYPE=MIXTURE or
  ALGORITHM=INTEGRATION.  Setting is ignored.
   1 WARNING(S) FOUND IN THE INPUT INSTRUCTIONS




SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         500

Number of dependent variables                                    4
Number of independent variables                                  0
Number of continuous latent variables                            1

Observed dependent variables

  Binary and ordered categorical (ordinal)
   A1          A2          A3          A4

Continuous latent variables
   F1


Estimator                                                       ML
Information matrix                                        OBSERVED
Optimization Specifications for the Quasi-Newton Algorithm for
Continuous Outcomes
  Maximum number of iterations                                 100
  Convergence criterion                                  0.100D-05
Optimization Specifications for the EM Algorithm
  Maximum number of iterations                                 500
  Convergence criteria
    Loglikelihood change                                 0.100D-02
    Relative loglikelihood change                        0.100D-05
    Derivative                                           0.100D-02
Optimization Specifications for the M step of the EM Algorithm for
Categorical Latent variables
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
Optimization Specifications for the M step of the EM Algorithm for
Censored, Binary or Ordered Categorical (Ordinal), Unordered
Categorical (Nominal) and Count Outcomes
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
  Maximum value for logit thresholds                            15
  Minimum value for logit thresholds                           -15
  Minimum expected cell size for chi-square              0.100D-01
Optimization algorithm                                         EMA
Integration Specifications
  Type                                                    STANDARD
  Number of integration points                                  15
  Dimensions of numerical integration                            1
  Adaptive quadrature                                           ON
Link                                                         LOGIT
Cholesky                                                        ON

Input data file(s)
  grm_dt.csv
Input data format  FREE


UNIVARIATE PROPORTIONS AND COUNTS FOR CATEGORICAL VARIABLES

    A1
      Category 1    0.456          228.000
      Category 2    0.076           38.000
      Category 3    0.468          234.000
    A2
      Category 1    0.422          211.000
      Category 2    0.146           73.000
      Category 3    0.432          216.000
    A3
      Category 1    0.442          221.000
      Category 2    0.122           61.000
      Category 3    0.436          218.000
    A4
      Category 1    0.450          225.000
      Category 2    0.096           48.000
      Category 3    0.454          227.000



THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       12

Loglikelihood

          H0 Value                       -1837.245

Information Criteria

          Akaike (AIC)                    3698.489
          Bayesian (BIC)                  3749.065
          Sample-Size Adjusted BIC        3710.976
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit for the Binary and Ordered Categorical
(Ordinal) Outcomes

          Pearson Chi-Square

          Value                             67.041
          Degrees of Freedom                    68
          P-Value                           0.5101

          Likelihood Ratio Chi-Square

          Value                             72.851
          Degrees of Freedom                    68
          P-Value                           0.3216



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 F1       BY
    A1                 0.893      0.161      5.549      0.000
    A2                 1.852      0.354      5.235      0.000
    A3                 1.078      0.181      5.961      0.000
    A4                 1.394      0.233      5.991      0.000

 Means
    F1                 0.000      0.000    999.000    999.000

 Thresholds
    A1$1              -0.207      0.105     -1.968      0.049
    A1$2               0.149      0.105      1.424      0.154
    A2$1              -0.495      0.150     -3.308      0.001
    A2$2               0.430      0.147      2.927      0.003
    A3$1              -0.293      0.112     -2.618      0.009
    A3$2               0.315      0.112      2.809      0.005
    A4$1              -0.282      0.124     -2.282      0.023
    A4$2               0.244      0.123      1.980      0.048

 Variances
    F1                 1.000      0.000    999.000    999.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.103E-01
       (ratio of smallest to largest eigenvalue)


RESULTS IN PROBABILITY SCALE

                    Estimate

 A1
    Category 1         0.456
    Category 2         0.076
    Category 3         0.468
 A2
    Category 1         0.422
    Category 2         0.146
    Category 3         0.432
 A3
    Category 1         0.441
    Category 2         0.122
    Category 3         0.437
 A4
    Category 1         0.449
    Category 2         0.096
    Category 3         0.456


     Beginning Time:  19:56:31
        Ending Time:  19:56:32
       Elapsed Time:  00:00:01



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2021 Muthen & Muthen
